docs/getting-started/development.md

```diff
@@ -63,154 +70,23 @@ The above may take some time to complete. Once the above commands complete, the
-
-// Behat testing environment
-$CFG->behat_prefix        = 'b_';
-$CFG->behat_dataroot      = dirname($CFG->dataroot) . '/behat';
-$CFG->behat_faildump_path = dirname($CFG->dataroot) . '/behat-faildump';
-$CFG->behat_wwwroot       = 'http://192.168.120.50/behat';
-$CFG->behat_profiles = array(
-    'chrome' => array(
-        'extensions' => array(
-            'Behat\MinkExtension\Extension' => array(
-                'selenium2' => array(
-                    'browser'     => 'chrome',
-                    'browserName' => 'chrome',
-                ),
-            ),
-        ),
-    ),
-    'firefox' => array(
-        'extensions' => array(
-            'Behat\MinkExtension\Extension' => array(
-                'selenium2' => array(
-                    'browser'     => 'firefox',
-                    'browserName' => 'firefox',
-                ),
-            ),
-        ),
-    ),
-    'iexplore' => array(
-        'extensions' => array(
-            'Behat\MinkExtension\Extension' => array(
-                'selenium2' => array(
-                    'browser'     => 'iexplore',
-                    'browserName' => 'iexplore',
-                ),
-            ),
-        ),
-    ),
-);
-$CFG->behat_config = array_merge(array(
-    'default' => array(
-        'extensions' => array(
-            'Behat\MinkExtension' => array(
-                'selenium2' => array(
-                    'wd_host' => "http://192.168.120.100:4444/wd/hub",
-                    'capabilities' => array(
-                        'browserVersion'    => 'ANY',
-                        'deviceType'        => 'ANY',
-                        'name'              => 'ANY',
-                        'deviceOrientation' => 'ANY',
-                        'ignoreZoomSetting' => 'ANY',
-                        'version'           => 'ANY',
-                        'platform'          => 'ANY',
-                    ),
-                ),
-            ),
-        ),
-    ),
-), $CFG->behat_profiles);
-
-// Selenium node path configuration
-// Requires https://github.com/moodle/moodle/compare/master...LukeCarrier:MDL-NOBUG-selenium-remote-node-file-upload-master
-$CFG->behat_node_dirroot = '/var/lib/selenium/moodle';
-$CFG->behat_node_dir_sep = '/';
-
-// PHPUnit testing environment
-$CFG->phpunit_prefix   = 'phpu_';
-$CFG->phpunit_dataroot = dirname($CFG->dataroot) . '/phpunit';
-
-// Bootstrap Moodle
-require_once __DIR__ . '/lib/setup.php';
+Copy and paste [this config file](development-config.php) into ``../Moodle/config.php``; then, perform
+
 ```
+# Sync your local sandbox to the Vagrant server
+$ vagrant rsync app-debug-1
+# Simulate a deployment
+$ vagrant ssh app-debug-1 --command 'sudo /usr/local/ubiquitous/bin/ubiquitous-set-current-release -d dev.local -r vagrant'
+# Verify
+$ vagrant ssh app-debug-1 --command 'sudo less /home/ubuntu/current/config.php'
+``` 
+ 
+Visiting [your development environment](http://192.168.120.50) should then succeed.
+
```

Comment:
This should happen as part of the Vagrant provisioning -- Vagrant assumes that all `synced_folder`s are required for provisioning and synchronises them. If this isn't happening for you, there may be a local configuration issue causing this?